redirects_generator/definitions.json

```
@@ -266,6 +266,16 @@
             "SourceUrl": "/dotnet/desktop/winforms/advanced/how-to-complete-windows-forms-print-jobs?view=netframeworkdesktop-4.8",
             "TargetUrl": "/dotnet/desktop/winforms/printing/overview?view=netdesktop-6.0"
         },
+        {
+            "Redirect": "TwoWay",
+            "SourceUrl": "/dotnet/desktop/winforms/advanced/how-to-print-a-windows-form?view=netframeworkdesktop-4.8",
+            "TargetUrl": "/dotnet/desktop/winforms/printing/how-to-print-windows-form?view=netdesktop-6.0"
+        },
+        {
+            "Redirect": "TwoWay",
+            "SourceUrl": "/dotnet/desktop/winforms/advanced/how-to-print-a-multi-page-text-file-in-windows-forms?view=netframeworkdesktop-4.8",
+            "TargetUrl": "/dotnet/desktop/winforms/printing/how-to-print-a-text-document?view=netdesktop-6.0"
```

Whoops, I made a mistake.
```suggestion
            "TargetUrl": "/dotnet/desktop/winforms/printing/how-to-print-text-document?view=netdesktop-6.0"
```